Expensify/App
# 1037 Chat / Allow charts originating in Google to be copied and pasted directly into Chat

### Details
Adding handler for when google charts MIME type is found in the paste event.
Handler converts the embedded base64 image into a Blobb in a File type to be passed to the modal. 

It is not possible to copy charts in the iOS and Android native apps. Hence I have left the below tests for iOS and Android unticked. The expected pattern for sharing a chart is using the native share functions in the OS.

I did have a short investigation on iOS simulator. When I [pasted on the iOS simulator](https://codepen.io/tech_query/pen/MqGgap) it didn't have any of the extra google specific MIME items present in the event. It only gave the text/html item, which would make detection more challenging. but I stopped this line of investigation due to the above reasons.

### Fixed Issues

Fixes https://github.com/Expensify/Expensify.cash/issues/1037

### Tests
1. Create a chart in google sheets. Details are in above issue.
2. Paste the chart into a chat text box.
3. Notice that the chart appears in the modal as an image.

I also tested pasting in images files to check regression. these still paste in as normal.

### Tested On


### Screenshots
![image](https://user-images.githubusercontent.com/47184118/103417742-d453e900-4b83-11eb-8a7f-9ecaacf45a1a.png)
